File: swift/Tests/ProjectNameSDKTests/ExistsTest.swift
// Univec SDK exists test.

import XCTest

@testable import UnivecSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = UnivecSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}

File: .sdk/tm/kotlin/test/ExistsTest.kt
package KOTLINPACKAGE.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import KOTLINPACKAGE.core.UnivecSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = UnivecSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
